cypress/e2e/draw/draw-feat-style.cy.ts

```diff
@@ -0,0 +1,363 @@
+      })
+  })
+
+  it('Validate Style and check that new feature style is encoded in URL', () => {
+    // URL is updated when clicking validation button
+    cy.get('button[data-cy="featureEditValidate"]').click()
+    cy.url().as('url')
+    cy.window()
+      .its('featureHash')
+      .then(function (fh) {
+        const features = new URLSearchParams(new URL(this.url).search).get(
+          'features'
+        )!
+        const ff = fh.readFeatures(features)
+        cy.wrap(ff).its('length').should('equal', 1)
+
+        cy.wrap(ff[0].getGeometry()?.getType()).should('equal', 'Point')
+        // check color green
+        cy.wrap(ff[0].get('c')).should('equal', '%2300ff00')
+        // check shape: cross
+        cy.wrap(ff[0].get('s')).should('equal', 'cross')
+        // check size 50
+        cy.wrap(ff[0].get('t')).should('equal', '50')
+
+        return cy.wrap(ff)
+      })
+      .its('length')
+      .should('equal', 1) // only one geometry has been created
+  })
+  it('Check that new feature style is applied to feature on map', () => {
+    cy.window()
+      .its('olMap')
+      .then(function (olMap) {
+        const featureLayers = olMap
+          .getLayers()
+          .getArray()
+          .filter((l: any) => l.get('cyLayerType') === 'featureLayer')
+        const features = featureLayers
+          .map((l: any) => l.getSource().getFeatures())
+          .flat()
+        cy.wrap(features.length).should('equal', 1)
+
+        // const ff = features.find((f: any) => f.get('name') === 'Point 1')
+        const ff = features[0]
+        cy.wrap(ff.featureType).should('equal', 'drawnPoint')
+        // check color green
+        cy.wrap(ff.featureStyle.color).should('equal', '#00ff00')
+        // check shape: cross
+        cy.wrap(ff.featureStyle.shape).should('equal', 'cross')
+        // check size = 50
+        cy.wrap(ff.featureStyle.size).should('equal', 50)
+        // check angle = 30°
+        // no idea why the angle setting cannot be checked here
+        // cy.log(ff.featureStyle.angle, (ff.featureStyle.angle / Math.PI) * 180)
+        // cy.wrap(Math.abs(30 - ff.featureStyle.angle / Math.PI * 180)).should('be.lessThan', 0.0001)
+      })
+  })
+
+  it('Cancel style changes and check that initial style is restored', () => {
+    cy.get('button[data-cy="featureEditCancel"]').click()
+
+    cy.window()
+      .its('olMap')
+      .then(function (olMap) {
+        const featureLayers = olMap
+          .getLayers()
+          .getArray()
+          .filter((l: any) => l.get('cyLayerType') === 'featureLayer')
+        const features = featureLayers
+          .map((l: any) => l.getSource().getFeatures())
+          .flat()
+        cy.wrap(features.length).should('equal', 1)
+        const ff = features[0]
+        // check size = 10
+        cy.wrap(ff.featureStyle.size).should('equal', 10)
+      })
+  })
+})
+
+// describe('Test style edition of Point feature', () => {
```

Comment:
Did you forget to uncomment or delete this `describe` ?

Reply:
oops